src/@utils/url.ts

```diff
@@ -1,5 +1,11 @@
+import * as isIPFS from 'is-ipfs'
+
 export function sanitizeUrl(url: string) {
   const u = decodeURI(url).trim().toLowerCase()
   const isAllowedUrlScheme = u.startsWith('http://') || u.startsWith('https://')
   return isAllowedUrlScheme ? url : 'about:blank'
 }
+
+export function isCID(value: string) {
```

Comment:
1. this seems overengineered, as it is pretty simple calling `isIPFS.cid(value)`
2. IPFS does not deal with URLs, but CIDs. Hence this method should not be in `@utils/url`, or the file need to be renamed